ui/src/components/search/services/searchService.js

```diff
@@ -29,9 +29,9 @@ export const searchVariants = async (inputValue) => {
     It's been modified to remove the size limit (can be added in if speed issues arise) and now also returns primary_site
     This can be modified further to allow a custom list of model properties to return in the search results.
 */
-export const searchModels = async (inputValue) => {
+export const searchModels = async (inputValue, apiFetcher) => {
   const query =
-    'query ModelsQuickSearchResults($sqon: JSON) {\n          models {\n            hits(filters: $sqon) {\n              total\n              edges {\n                node {\n                  primaryKey: name\n                  autocomplete: autocomplete\n primary_site\n                }\n              }\n            }\n          }\n        }';
+    'query ModelsQuickSearchResults($sqon: JSON) {\n          model {\n            hits(filters: $sqon) {\n              total\n              edges {\n                node {\n                  primaryKey: name\n                  autocomplete: autocomplete\n primary_site\n                }\n              }\n            }\n          }\n        }';
```

Comment:
in a "leave things better than you found them" spirit, this query could be spaced out in a more readable manner (e.g. actual line breaks instead of coded ones, etc.)
e.g. https://github.com/overture-stack/arranger/blob/main/modules/components/src/DataContext/dataQueries.ts#L47